### DISCUSSION TITLE
Table settings for each user but not common

### DISCUSSION THREAD

**Discussion by stepanof** [nocodb/nocodb#3068] [Q&A] [answered]
Hello team.

When I'm working with table via nocodb UI I use some filters. Im not the only nocodb user in our team so other users see table with my custom filters when open the same table as me.

Because of it there is no possibility to work with the same table by 2+ users at the same moment without confusion. 
Users want to apply their own filters locally but not globally. 

Is it possible to change table settings behaviour in current version?
If not I create a new discussion with feature request.

![image](https://user-images.githubusercontent.com/68236577/183920424-c672f64b-bd10-4151-80b8-467095df59f7.png)

Best regards,
Sergey Stepanov

**Comment by o1lab** [comment, marked as answer]
At the moment is not possible, you can possibly 
- take a copy of the view 
- prefix a your name in front of the view & play around with that.

**Reply by stepanof** [reply]
Thanks!

**Comment by cheetahfish** [comment]
This is a big pain for our use. Users don't understand that they are changing filters for everyone. This is super annoying as I then have to go and lock and reset the filters. and then duplicate views to just specific users.

**Reply by nquijano** [reply]
Wholeheartedly agree... Having sort, group and filters settings saved client side and 100% configurable even in anonymous read only views would absolutely make me more than happy. 
Keep up the hard work devteam !
